lib/process-services-cloud/src/lib/form/mocks/attach-file-cloud-widget.mock.ts

```diff
@@ -0,0 +1,328 @@
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: {
+            value: null,
+            name: 'stringVariableName',
+            type: DestinationFolderPathType.STRING_TYPE
+        }
+    }
+};
+
+export const allSourceWithFolderTypeEmptyValue = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: {
+            value: null,
+            name: 'folderVariableName',
+            type: DestinationFolderPathType.FOLDER_TYPE
+        }
+    }
+};
+
+export const allSourceWithRootParams = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: {
+            value: '-root-',
+            type: DestinationFolderPathType.STATIC_TYPE
+        }
+    }
+};
+
+export const allSourceWithWrongAliasParams = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: {
+            name: 'staticValue',
+            value: '-wrongAlias-',
+            type: DestinationFolderPathType.STATIC_TYPE
+        }
+    }
+};
+
+export const allSourceWithNoAliasParams = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: {
+            name: 'staticValue',
+            value: '/noalias/createdFolder',
+            type: DestinationFolderPathType.STATIC_TYPE
+        }
+    }
+};
+
+export const allSourceWithoutDestinationFolderPath = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID
+    }
+};
+
+export const allSourceWithoutValueProperty = {
+    fileSource: {
+        name: 'all file sources',
+        serviceId: FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID,
+        destinationFolderPath: '-mockAlias-'
+    }
+};
+
+export const fakeMinimalNode: Node = <Node> {
+    id: 'fake',
+    name: 'fake-name',
+    content: {
+        mimeType: 'application/pdf'
+    }
+};
+
+export const fakeNodeWithProperties: Node = <Node> {
+    id: 'fake-properties',
+    name: 'fake-properties-name',
+    content: {
+        mimeType: 'application/pdf'
+    },
+    properties: {
+        'pfx:property_one': 'testValue',
+        'pfx:property_two': true
+    }
+};
+
+export const expectedValues = {
+    pfx_property_one: 'testValue',
+    pfx_property_two: true
+};
+
+export const mockNodeId = new Promise(function(resolve) {
+    resolve('mock-node-id');
+});
+
+export const mockNodeIdBasedOnStringVariableValue = new Promise(function(resolve) {
+    resolve('mock-string-value-node-id');
+});
+
+export const mockNodeIdBasedOnFolderVariableValue = new Promise(function(resolve) {
+    resolve('mock-folder-value-node-id');
+});
+
+export const fakeLocalPngAnswer = {
```

Comment:
can we name it fakeLocalPngResponse instead of Answer?

Reply:
Updated. Thanks